packages/react/src/CTABanner/CTABanner.module.css

```diff
@@ -0,0 +1,76 @@
+.CTABanner {
+  position: relative;
+  max-width: 1280px;
+  margin: 0 auto;
+  display: flex;
+  gap: var(--base-size-16);
+  flex-direction: column;
+}
+
+.CTABanner--Shadow {
+  position: absolute;
+  transform: translate(-50%, 0%);
+  width: calc(60% + 60px);
+  height: calc(100% + 60px);
+  border-radius: 50% 22% 40% 80%;
+  filter: blur(50px);
+  top: 0%;
+  opacity: 0.7;
+}
+
+.CTABanner--RightShadow {
+  left: 30%;
+  background: radial-gradient(circle at 50% 50%, rgba(26, 127, 55, 1), rgba(26, 127, 55, 0));
+}
+
+.CTABanner--LeftShadow {
+  left: 70%;
+  background: radial-gradient(circle at 50% 50%, rgba(3, 102, 214, 1), rgba(3, 102, 214, 0));
```

Comment:
We should use the css variables here if possible, which would also apply the correct values for light **_and_** dark modes (guessing these are light?)

![gradients for light and dark mode](https://user-images.githubusercontent.com/13340707/202435446-1b73b4ba-7dd4-4d7e-afeb-a0f9f25f6f6e.png)

For now, I think you can resolve this by swapping to the gradient tokens directly, and in a separate PR we can create the required tokens for all gradients and backfill.

Reply:
All set thanks for the help here